### PR TITLE
(WSR 8) - <ModalSelectorLayout/> - use button unidriver

### PR DESCRIPTION
**Pull request by EgoziE**
Use `<Button/>` unidriver in `<ModalSelectorLayout/>` driver

### CODE REVIEW

**Comment by EgoziE** [comment]
tests failures are unrelated

File: src/ModalSelectorLayout/ModalSelectorLayout.protractor.driver.js [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
-    getButtonTextContent: () => element.getText(),
-    isButtonDisabled: () => hasAttribute(element, 'disabled'),
-    isPrefixIconExists: () => element.$('[data-hook="btn-prefix"]').isPresent(),
-    isSuffixIconExists: () => element.$('[data-hook="btn-suffix"]').isPresent(),
-    element: () => element,
-  };
-
-  return mergeDrivers(publicDriver, focusableDriver);
-};
+import { buttonDriverFactory } from '../Button/Button.uni.driver';
```
Comment (mykas) [inline src/ModalSelectorLayout/ModalSelectorLayout.protractor.driver.js]: Actually our uni driver does not have  all the methods that were previously exposed. These methods are not present in current uni driver:
```
isPrefixIconExists: () => element.$('[data-hook="btn-prefix"]').isPresent(),	
isSuffixIconExists: () => element.$('[data-hook="btn-suffix"]').isPresent(),	
element: () => element,
```

- `element` method should be definitely.
- `isPrefixIconExists` and `isSuffixIconExists` also should be deprecated as we encourage our users to put their own data-hook on their elements and test it.

Comment (EgoziE) [reply src/ModalSelectorLayout/ModalSelectorLayout.protractor.driver.js]: `isPrefixIconExists` and `isSuffixIconExists` are deprecated and were removed in another PR so it's ok
and `element` function exists in `baseUniDriverFactory` :)

Comment (mykas) [reply src/ModalSelectorLayout/ModalSelectorLayout.protractor.driver.js]: oh, right `element` is from baseDriver

**Pull request merged**